addons/website/static/src/components/navbar/navbar.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-inherit="web.NavBar" t-inherit-mode="extension" owl="1">
+    <xpath expr="//div[hasclass('o_menu_systray')]" position="attributes">
+        <attribute name="t-att-class">isWebsiteSystray ? 'o_website_systray' : ''</attribute>
+    </xpath>
+</t>
+
+</templates>
```

Comment:
Not a fan of this (if another app does the same, everything is broken) and still think it should not be necessary (there should not be website specific design here).

And if it was necessary, this should be squashed IMO.

Reply:
Squashed & used `add=` to avoid anything possibly broken